### PR TITLE
Qingwei/fix resubscribe bug

### PR DESCRIPTION
**Pull request by qingweibinary**
Not tested, better dun merge

### CODE REVIEW

**Comment by qingweibinary** [comment]
@borisyankov Tested, should fixed those api errors when change languages

File: src/LiveApi.js [hunk 1]
```diff
@@ -125,10 +130,7 @@ export default class LiveApi {
     }
 
     onClose() {
-        setTimeout(() => {
-            this.connect();
-            this.resubscribe();
-        }, 1000);
+        this.connect();
```
Comment (qingweibinary) [inline src/LiveApi.js]: Attention ~
Not sure why it was delayed, thus remove, hope this is fine

File: src/LiveError.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 export default class LiveError extends Error {
     constructor(errorObj) {
         super();
-        this.message = `Server Error: (${errorObj.code}) ${errorObj.message}`;
+        this.message = `Server Error: (${errorObj.code}) ${errorObj.message}, caused by ${errorObj.echo_req}`;
```
Comment (borisyankov) [inline src/LiveError.js]: Good idea, I implemented it more thoroughly in another commit ;)

File: src/LiveApi.js (hunk 1 above)
Comment (borisyankov) [inline src/LiveApi.js]: Was intentional :)
1sec timeout on disconnect.
On the other hand, it was totally not tested in real scenario, and if no delay works too, is better for sure ;)

Comment (qingweibinary) [inline src/LiveApi.js]: I tested with no delay, working fine, and I think we need to have as little delay as possible (Still handshake is taking time)

**Pull request merged**